source/PlayState.hx

```diff
@@ -5,23 +5,35 @@ import flixel.FlxState;
 import flixel.graphics.FlxGraphic;
 import flixel.tile.FlxTilemap;
 import flixel.util.FlxColor;
+import flixel.util.FlxTimer;
 
 class PlayState extends FlxState
 {
 	public var GRAVITY(default, never):Float = 600;
 	
 	private var map:FlxTilemap;
 	private var player:Player;
+	private var flagpole:FlagPole;
 	public static var hud:HeadsUpDisplay;
-	
+
 	override public function create():Void
 	{
-		if (hud == null){
-			hud = new HeadsUpDisplay(0, 0, "MARIO");
-		}
+		//if (hud == null){
+		hud = new HeadsUpDisplay(0, 0, "MARIO");
+		//}
 		super.create();
 		
-		player = new Player(50, 50);
+		/*Create the flagpole at the end of the level 
+		 * This will also instantiate the flag
+		*/
+		//544, 352
+		flagpole = new FlagPole(544, 352);
```

Comment:
added flag_x_loc and flag_y_loc as variables.
Changed flagpole declaration to:
flagpole = new FlagPole(32*flag_x_loc, 32*flag_y_loc);

flag_x_loc is the number of blocks to the right where we want the flag
flag_y_loc is the number of blocks down we want the flag